docs/GettingStartedDocs/Contributors/WindowsManualSGX1Prereqs.md

```diff
@@ -1,49 +1,58 @@
+
+1. Download Intel SGX PSW from [Windows Open Enclave SDK Prerequisites](WindowsPrerequisites.md). For this example we will use Intel SGX PSW for Windows v2.12.100.4, but your commands should substitute the version with the version downloaded.
+
+2. Unpack the self-extracting ZIP executable, and run the installer under `PSW_EXE_RS2_and_before`:
 
 ```cmd
 "C:\Intel SGX PSW for Windows v2.12.100.4\PSW_EXE_RS2_and_before\Intel(R)_SGX_Windows_x64_PSW_2.12.100.4.exe"
 ```
 
-On Windows 10 and Windows Server 2019, you should ensure that you have the latest drivers
-by checking for updates and installing all necessary updates.
-
-If you would like to manually update the PSW on Windows Server 2019 or Windows
-clients > 1709 without relying on Windows Update, you can update the PSW components
-as follows:
+### Manual installation for client versions 1709 and above
+If you would like to manually update Intel PSW on Windows Server 2019 or Windows clients >= 1709 without relying on Windows Update, you can update the PSW components as follows:
 
-1. Install `devcon.exe`, available as part of the [Windows Driver Kit for Windows 10](https://go.microsoft.com/fwlink/?linkid=2026156).
+1. Download devcon from [Windows Open Enclave SDK Prerequisites](WindowsPrerequisites.md) and install. Alternatively it is available as part of the [Windows Driver Kit for Windows 10](https://go.microsoft.com/fwlink/?linkid=2026156).
    -  Note that `devcon.exe` is usually installed to `C:\Program Files (x86)\Windows Kits\10\tools\x64`
    which is not in the `PATH` environment variable by default.
 
-2. In an elevated command prompt, run the following command from the extracted PSW package under the `PSW_INF_RS3_and_above` folder:
+2. Download Intel SGX PSW from [Windows Open Enclave SDK Prerequisites](WindowsPrerequisites.md).
+
+3. Unpack the self-extracting ZIP executable. In an elevated command prompt, run the following command from the extracted PSW package under the `PSW_INF_RS3_and_above` folder:
   ```cmd
   devcon.exe update sgx_psw.inf "SWC\VEN_INT&DEV_0E0C"
   ```
 
+### Verifying PSW installation
 You can verify that the correct version of Intel SGX PSW is installed by using
 Windows Explorer to open `C:\Windows\System32`. You should be able to find
 file `sgx_urts.dll` if PSW is installed. Right click on `sgx_urts.dll`,
 choose `Properties` and then find `Product version` on the `Details` tab.
-The version should be "2.12.xxx.xxx" or above.
+The version should be `2.12.xxx.xxx` or above.
 
 To verify that Intel SGX PSW is running, use the following command:
 
 ```cmd
 sc query aesmservice
 ```
 
-The state of the service should be "running" (4). Follow Intel's documentation for
-troubleshooting. In case the AESM service was stopped for some reasons, restart it
-using the following command from Powershell.
+The state of the service should be "running" (4). If there are any errors, follow Intel's documentation for
```

Comment:
can "documentation" be hotlinked to some actual document to follow?

Reply:
Unfortunately Intel does not publish the documentation directly, and is instead included with the installation files. I will add a reference to where it can be found within the installation files.